### ISSUE TITLE
List of people who unfriendship me

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Robyer/robyer#5]
```
Sorry for the bad english, i've recently changed jarvis facebook protocol for 
yours and when i open miranda start a long series of system popup windows 
(besides miranda notifications) of people who unfriendship me in facebook.
I've forced to close miranda because that pop ups doesn't stops.
Is there some other way to check who unfriendship me (like msn miranda's 
contact list in options >> protocol >> msn >> server list) or something?
Thanks, keep good work.
Ps this is a function that you should not make very public, i think faceb@@k 
assh@oles doesn't like...
```

Original issue reported on code.google.com by `ricardo...@gmail.com` on 12 Sep 2011 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
Hi,
1) This function is not some "hacking" of facebook, but just how miranda works 
:)
 - point is, that on login it checks all contacts which are on facebook server with that which are saved in miranda profile. Then only notifies that contacts which are in miranda but not on server.

2) That popup notification gets only once for each contact - to contact is 
added db key "Deleted" with value "1", so if you want to find these contacts 
again, you just should use Search function of Database Editor ++ plugin.

You should delete that deleted contacts from Miranda by yourself, plugin keeps 
them there e.g. because saved history with them :)
```

Original comment by `ROBYER` on 13 Sep 2011 at 1:51

**Comment by GoogleCodeExporter** [comment]
Original comment by `ROBYER` on 14 Sep 2011 at 4:40
- Changed state: **WontFix**
